test/performance/load-test/sla.go

```diff
@@ -63,6 +64,32 @@ func newLoadTestMaximumLatency(tags ...string) *tpb.ThresholdAnalyzerInput {
 	}
 }
 
+// This analyzer validates that the mean error rate observed over the 0->3k
+// stepped burst is no more than +1 than the historic runs. The settings, e.g.
+// window size, tolerance params, are derived mainly base on
+// https://github.com/google/mako/blob/master/docs/WDA_ANALYZER.md#const-term
+func newLoadTestMaximumErrorRate(tags ...string) *wpb.WindowDeviationInput {
+	return &wpb.WindowDeviationInput{
+		Name: proto.String("Mean error rate"),
+		RunInfoQueryList: []*mpb.RunInfoQuery{{
+			Tags:  tags,
+			Limit: proto.Int32(50),
+		}},
+		ToleranceCheckList: []*wpb.ToleranceCheck{{
+			RecentWindowSize:            proto.Int32(7),
+			MinimumHistoricalWindowSize: proto.Int32(50),
+			DirectionBias:               wpb.ToleranceCheck_IGNORE_DECREASE.Enum(),
+			DataFilter: &mpb.DataFilter{
+				DataType: mpb.DataFilter_METRIC_AGGREGATE_MEAN.Enum(),
+				ValueKey: proto.String("es"),
+			},
+			MedianToleranceParamsList: []*wpb.MedianToleranceParams{{
+				ConstTerm: proto.Float64(1),
```

Comment:
So does this permit creep every day of 1error more until we're 100% errors?
I.e. do we need an absolute one as well (unless we already do, and then this is moot point).

Reply:
That's a good point, thanks.
I rechecked the current benchmark runs, the error rates are mostly zero (except some noises). I think we should be able to use a ThresholdAnalyzer with CrossRunConfig to validate it's zero, instead of using a WindowDeviationAnalyzer.